elephant/asset.py

```diff
@@ -267,70 +267,58 @@ def intersection_matrix(
+    t_start_x, t_start_y : pq.Quantity, optional
+        The start time of the binning for the first and second axes of the intersection
         matrix, respectively.
-        If None (default) the attribute t_start of the SpikeTrains is used
+        If None (default) the attribute `t_start` of the SpikeTrains is used
         (if the same for all spike trains).
         Default: None
-    t_start_y : pq.Quantity, optional
-        start time of the binning for the second axis of the intersection
-        matrix
-        If None (default) the attribute t_start of the SpikeTrains is used
-        (if the same for all spike trains).
-        Default: None
-    t_stop_x : pq.Quantity, optional
-        stop time of the binning for the first axis of the intersection
+    t_stop_x, t_stop_y : pq.Quantity, optional
+        The stop time of the binning for the first and second axes of the intersection
         matrix, respectively.
         If None (default) the attribute t_stop of the SpikeTrains is used
         (if the same for all spike trains).
         Default: None
-    t_stop_y : pq.Quantity, optional
-        stop time of the binning for the second axis of the intersection
-        matrix
-        If None (default) the attribute t_stop of the SpikeTrains is used
-        (if the same for all spike trains).
-        Default: None
     norm : int, optional
-        type of normalization to be applied to each entry [i,j] of the
-        intersection matrix. Given the sets s_i, s_j of neuron ids in the
-        bins i, j respectively, the normalisation coefficient can be:
+        The normalization type to be applied to each entry `M[i,j]` of the
+        intersection matrix `M`. Given the sets `s_i`, `s_j` of neuron ids in the
+        bins `i`, `j` respectively, the normalisation coefficient can be:
 
             * norm = 0 or None: no normalisation (row counts)
-            * norm = 1: len(intersection(s_i, s_j))
-            * norm = 2: sqrt(len(s_1) * len(s_2))
-            * norm = 3: len(union(s_i, s_j))
+            * norm = 1: `len(intersection(s_i, s_j))`
+            * norm = 2: `sqrt(len(s_1) * len(s_2))`
+            * norm = 3: `len(union(s_i, s_j))`
 
         Default: None
 
     Returns
     -------
-    imat : np.ndarray of float
-        the intersection matrix of a list of spike trains. Has shape (n,n),
-        where n is the number of bins time was discretized in.
-    x_edges : np.ndarray
-        edges of the bins used for the horizontal axis of imat. If imat is
-        a matrix of shape (n, n), x_edges has length n+1
-    y_edges : np.ndarray
-        edges of the bins used for the vertical axis of imat. If imat is
-        a matrix of shape (n, n), y_edges has length n+1
+    imat : (n,n) np.ndarray
+        The intersection matrix (of type float) of a list of spike trains. Has
+        the shape (n,n), where `n` is the number of bins time was discretized in.
+    x_edges : (n,n) np.ndarray
```

Comment:
Inside the function, `x_edges` and `y_edges` are named `xx` and `yy` respectively. According to our conventions, we should use the same name as in the code, but I guess refactoring the names inside the function would add to clarity.